### ISSUE TITLE
keyboard does not work for navigation

### ISSUE THREAD

**Issue by andreva7** [MoneyTools/MoneyFlutter#6]
Select an account. 
Click transactions to see the list of transactions in that account
I cannot use the up\down keys to scroll through the transactions. keyboard focus remains on the account list

**Comment by clovett** [comment]
Can you explain this step: "Click transactions to see the list of transactions in that account."?  I normally just "select an account" that is enough for those transactions to show up.  But yes, it does not automatically move the focus to the transactions view.  in .NET WPF apps the "default behavior of keyboard focus" is:

1. Whatever view you mouse down in gets the keyboard focus
2. The keyboard focus stays there until you type TAB which can move to the "next control" in a "given TAB order".

The TAB order in the Account/Categories/Payees/Securities selector on the left is to tab through each of those headings then the 5th tab will land you in the Transaction view where it will now have focus.

You can also find out what has the keyboard focus by typing ALT  twice and the control with keyboard focus often gets a white or black border.  The list views are not showing an overall "focus feedback" because in those views you can see the focus happening on the selected item instead.

**Comment by andreva7** [comment]
These are issues in JP's flutter UI, not the MyMoney UI.

-Andre
